src/sentry/models/rulesnooze.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.db.models import CheckConstraint, Q
+
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model, sane_repr
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_only_model
+class RuleSnooze(Model):
+    """
+    Duration an issue alert or metric alert is snoozed for a user. Null `until` value means snoozed forever.
+    """
+
+    __include_in_export__ = True
+
+    user_id = HybridCloudForeignKey("sentry.User", unique=True, on_delete="CASCADE")
+    rule = FlexibleForeignKey("sentry.Rule", null=True, unique=True)
```

Comment:
You'll also need to remove these `uniques` and just rely on the `together` ones

Reply:
oop